### PR TITLE
Update DG to include import use case

### PR DESCRIPTION
**Pull request by JasonCP14**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -406,6 +406,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+      Use case ends.
+
+* 1b. The given file has invalid value, incorrect format, and/or duplicate persons.
+
+    * 1b1. InternConnect shows an error message.
+
+      Use case ends.
+
+* 1b. The given file has invalid value, incorrect format, and/or duplicate persons.
```
Comment (maryjess) [inline docs/DeveloperGuide.md:433]: Is this duplicate intended?

**Review by maryjess — APPROVED** [review]
Just need a small tweak to erase the duplicate, after that LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (JasonCP14) [reply docs/DeveloperGuide.md:433]: It is not, will fix it in the next commit

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -406,6 +406,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Import applicants from an external json file**
+
+**MSS**
+
+1.  User requests to import applicants from a json file
+2.  InternConnect adds all applicants
```
Comment (cliftonfelix) [inline docs/DeveloperGuide.md:414]: Better to put 1. instead of 2.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -406,6 +406,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+3.  InternConnect shows the updated list of applicants
```
Comment (cliftonfelix) [inline docs/DeveloperGuide.md:415]: Same here

**Review by cliftonfelix — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (JasonCP14) [reply docs/DeveloperGuide.md:415]: No, it's perfectly fine.

**Pull request merged**